hosts/mno001/network.nix

```diff
@@ -37,33 +37,55 @@ in
         bondConfig = {
           Mode = "802.3ad"; # LACP 
           MIIMonitorSec = "250ms";
-          LACPTransmitRate = "fast";
+          LACPTransmitRate = "fast"; 
         };
       };
 
-      "20-svc-management" = {
+      # 20-svc-internet
+      "20-uplink" = {
```

Comment:
We could use a more generic naming for the virtual unnumbered subinterfaces? Something like `bond.100`.